adam/runtime/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN mkdir /opt/cgl-docker-lib
 COPY adam /opt/cgl-docker-lib/adam
 
 # copy spark
-COPY apache-spark/apache-spark /opt/cgl-docker-lib/apache-spark
+COPY apache-spark /opt/cgl-docker-lib/apache-spark
```

Comment:
Does this line keep going back and forth?  or maybe I've just seen it in more than one commit